src/pybind/mgr/dashboard_v2/tests/test_ping.py

```diff
@@ -8,20 +8,16 @@
 
 
 class PingTest(ControllerTestCase):
```

Comment:
Do we still need `PingTest` at all? I'd remove all echo and ping controllers now.

Reply:
yes we can remove it